go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/google/uuid v1.1.2 // indirect
 	github.com/googleapis/gnostic v0.5.5 // indirect
+	github.com/gophercloud/gophercloud v0.0.0-20221230131953-acb6bd9c1992 // indirect
```

Comment:
We don't update gophercloud herr. Please refer line 94.
And I think we don't need to update go.mod because current reference:
github.com/Wind-River/gophercloud v0.0.0-20230731150227-137664ed2caf
includes gophercloud update;
https://github.com/Wind-River/gophercloud/commit/acb6bd9c1992c0aa3901a8a82542339873698be8